qingstor.php

```diff
@@ -1,26 +1,24 @@
 <?php
 /*
 Plugin Name: WP-QingStor
-Plugin URI:  https://github.com/yungkcx/QingStor-Plugin-for-WordPress
-Description: QingStor Plugin for WordPress. The Backup function requires zip and mysqldump program.
+Plugin URI:  https://github.com/yunify/wp-qingstor
+Description: QingStor Plugin for WordPress, support auto sync Media Library and delete local files. This project is based on the secondary development of yunify/wp-qingstor.
```

Comment:
Remove `This project is based on the secondary development of yunify/wp-qingstor` please.

Reply:
Remove already.